_layouts/partners/faq.html

```diff
@@ -81,39 +15,39 @@ <h1 class="partners-title">{{ page.title }}</h1>
-                  <div class="usa-accordion usa-accordion--bordered margin-bottom-5" aria-multiselectable="true">
+                  <div class="usa-accordion usa-accordion--bordered margin-bottom-5">
                       {% for item in page.general_accordion %}
-                          <div class="usa-accordion__heading">
-                              <button
-                                onclick="updateSingleAccordion(this, 'general')"
-                                class="usa-accordion__button login-accordion-btn-general"
-                                aria-expanded="false"
-                                aria-controls={{ item.id }}
-                                >
-                                {{ item.title }}
-                              </button>
-                          </div>
-                          <div id={{ item.id }} class="usa-accordion__content login-accordion-content-general">
-                          {{ item.content | markdownify}}
-                          </div>
+                        <h3 class="usa-accordion__heading">
+                          <button
+                            onclick="updateSingleAccordion(this, 'general')"
+                            class="usa-accordion__button login-accordion-btn-general"
+                            aria-expanded="false"
+                            aria-controls={{ item.id }}
+                            >
+                            {{ item.title }}
+                          </button>
+                        </h3>
+                        <div id={{ item.id }} class="usa-accordion__content login-accordion-content-general">
+                        {{ item.content | markdownify}}
+                        </div>
                       {% endfor %}
                   </div>
-                  <a href="#main-content" class="usa-current">Back to Top</a>
+                  <a href="#main-content">Back to Top</a>
                   <div class="section-title-row margin-top-5">
                     <div id="logistics-section" class="section-title-left">{{ page.logistics_section_title | markdownify}}</div>
-                    <a 
-                      class="section-title-right collapse-expand-btn-logistics"
-                      href="javascript:updateAllAccordions('logistics')">
+                    <button
+                      class="section-title-right collapse-expand-btn-logistics usa-button usa-button--unstyled"
+                      onclick="javascript:updateAllAccordions('logistics')">
                       Expand all
-                    </a>
+                    </button>
                   </div>
-                  <div class="usa-accordion usa-accordion--bordered margin-bottom-5" aria-multiselectable="true">
+                  <div class="usa-accordion usa-accordion--bordered margin-bottom-5">
                       {% for item in page.logistics_accordion %}
                           <div class="usa-accordion__heading">
```

Comment:
Can we make this a heading tag as well, like we did above?

Same note applies for other `usa-accordion__heading` in this file.

Reply:
Ok - updates made.